webclient/js/elems/raflow.js

```diff
@@ -58,6 +58,41 @@ $(document).on('click', '#ra-form #progressbar a', function () {
     return false;
 });
 
+// lockOnGrid
+// Lock grid if chebox is unchecked(false). Unlock grid if checkbox is checked(true).
+// Lock grid when there is no record in the grid.
+window.lockOnGrid = function (gridName) {
+    console.log("lockOnGrid");
+    var isChecked = $("#" + gridName + "_checkbox")[0].checked;
+    var recordsLength = w2ui[gridName].records.length;
+    console.log(isChecked);
```

Comment:
remove/comment these console statements if those are working fine

Reply:
I need it for now.

Reply:
:) :+1: